ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/aaa-accounting.j2

```diff
@@ -1,22 +1,26 @@
-{% if aaa_accounting is defined and aaa_accounting is not none %}
+{% if aaa_accounting is arista.avd.defined %}
 
 ## AAA Accounting
 
 ### AAA Accounting Summary
 
 | Type | Sub-type | Record | Accounting Stores | Logging |
 | ---- | -------- | ------ |------------------ | ------- |
-{%     if aaa_accounting.exec.default is defined and aaa_accounting.exec.default is not none %}
-{%        if aaa_accounting.exec.default.group is defined and aaa_accounting.exec.default.group is not none %}
-| Exec | - | {{ aaa_accounting.exec.default.type }} | {{ aaa_accounting.exec.default.group }} | - |
-{%        endif %}
+{%     if aaa_accounting.exec.console.group is arista.avd.defined %}
```

Comment:
missing variable check for `aaa_accounting.exec.console.type`
Also missing variable checks below